MetaMask/metamask-extension
# Migrate the build script to `yargs`

## Explanation

The build script now uses `yargs` rather than `minimist`. The CLI is now better documented, and we have additional validation for each option.

A patch for `yargs` was required because it would blow up on the line `Error.captureStackTrace`. For some reason when running under LavaMoat, that property did not exist.

## More Information

Closes #12766

## Manual Testing Steps

Run `node ./development/build/index.js --help` and try out all of the options! Try running the build scripts in `package.json` as well to ensure they work (e.g. `yarn start`, `yarn dist`, etc.).

## Pre-Merge Checklist


### + If there are functional changes:
